python/ray/services.py

```diff
@@ -439,12 +442,18 @@ def start_ray_process(command,
         # version, and tmux 2.1)
         command = ["tmux", "new-session", "-d", "{}".format(" ".join(command))]
 
-    # Block sigint for spawned processes so they aren't killed by the SIGINT
-    # propagated from the shell on Ctrl-C so we can handle KeyboardInterrupts
-    # in interactive sessions. This is only supported in Python 3.3 and above.
-    def block_sigint():
+    if fate_share is None:
+        logger.warning("fate_share= should be passed to start_ray_process()")
```

Comment:
Hey quick question, on the current master, running just `ray.init` spits out a lot of this messages. Can we:
- Reduce the logger verbosity
- Or actually make this go away
```
2020-03-10 11:31:14,339	INFO resource_spec.py:212 -- Starting Ray with 33.01 GiB memory available for workers and up to 0.09 GiB for objects. You can adjust these settings with ray.init(memory=<bytes>, object_store_memory=<bytes>).
2020-03-10 11:31:14,339	WARNING services.py:456 -- fate_share= should be passed to start_ray_process()
2020-03-10 11:31:14,456	WARNING services.py:456 -- fate_share= should be passed to start_ray_process()
2020-03-10 11:31:14,572	WARNING services.py:456 -- fate_share= should be passed to start_ray_process()
2020-03-10 11:31:14,579	WARNING services.py:456 -- fate_share= should be passed to start_ray_process()
2020-03-10 11:31:14,686	WARNING services.py:456 -- fate_share= should be passed to start_ray_process()
2020-03-10 11:31:14,692	INFO services.py:1123 -- View the Ray dashboard at localhost:8265
2020-03-10 11:31:14,694	WARNING services.py:456 -- fate_share= should be passed to start_ray_process()
2020-03-10 11:31:14,700	WARNING services.py:456 -- fate_share= should be passed to start_ray_process()
2020-03-10 11:31:14,707	WARNING services.py:456 -- fate_share= should be passed to start_ray_process()
2020-03-10 11:31:14,714	WARNING services.py:456 -- fate_share= should be passed to start_ray_process()
```
(cc @edoakes )

Reply:
Yeah, it's already being addressed in #7545 :)